rebac-admin-backend/v1/error.go

```diff
@@ -8,46 +8,47 @@ import (
 	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
 )
 
-// UnauthorizedError represents unauthorized access error.
-type UnauthorizedError struct {
-	message string
-}
-
-func (e *UnauthorizedError) Error() string {
-	return fmt.Sprintf("Unauthorized: %s", e.message)
-}
+// ErrorCode is a code which describes the class of error.
+type ErrorCode string
 
-// NewUnauthorizedError returns a pointer to a new instance of UnauthorizedError with the provided message
-func NewUnauthorizedError(message string) *UnauthorizedError {
-	return &UnauthorizedError{message}
-}
+const (
+	CodeUnauthorized    ErrorCode = "unauthorized"
```

Comment:
godocs, please

Reply:
It's not there anymore. So I resolve this comment.